pkgs/development/tools/goimports/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, go, fetchurl, fetchgit, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  rev = "7534f4943d94a318edde90212439e538ed54cdde";
+  version = "${lib.strings.substring 0 7 rev}";
```

Comment:
Change `version` to a date of creation of that revision (in `YYYY-MM-DD` format).
See https://github.com/NixOS/nixpkgs/pull/6539